README.md

```diff
@@ -61,49 +61,38 @@ fdb-operator-76d6dc9df-dcvmg   1/1     Running   0          3m10s
 Command (for copy paste):
 
 ```
-kubectl apply -k tigris-deploy/fdb-cluster/base
+kubectl apply -k tigris-deploy/fdb-cluster/overlays/example
 ```
 
 Output:
 
 ```
-$ kubectl apply -k tigris-deploy/fdb-cluster/base
+$ kubectl apply -k tigris-deploy/fdb-cluster/overlays/example
 Warning: apps.foundationdb.org/v1beta1 FoundationDBCluster is deprecated; use apps.foundationdb.org/v1beta2 FoundationDBCluster
```

Comment:
Do we need to do anything about this warning?

Reply:
It's safe to ignore it at this point. IIRC the upgrade requires changes so it is not just a simple header rewrite to the newest API. I have logged an issue to get this upgraded cohesively everywhere!